### DISCUSSION TITLE
WSA open-source now?

### DISCUSSION THREAD

**Discussion by eerke2021** [microsoft/WSA#585] [General]
Is WSA open-source code yet? Seems a shame to spend all that time writing the code and let it disappear. I'd prefer it if Microsoft had maintained WSA but Open-sourcing the code to the community seems effortless and useful, assuming the code isn't embarrassingly bad or full of in-appropriate comments.

**Comment by GGuyX** [comment]
Even if the code was bad, it would still be worth having it. I loved to use WSA for the time that I did but I stopped when I heard it would be going out of support. It's great for mobile games on PC, even if it's pretty resource intensive. I really, REALLY hope they do make this open-source at the end of the day, though sadly I have my doubts.

**Comment by eerke2021** [comment]
I meant “bad” as in “embarrassingly bad” from their point of view. I was trying to think why Microsoft don’t just open source the code. I too would be happy with what I personally think is “bad” code. I have a fire-stick and have never bought anything from the Amazon store so I’m not surprised they closed a feature funded from the store. On 8 Aug 2024, at 14:19, GGuyX ***@***.***> wrote:﻿
Even if the code was bad, it would still be worth having it. I loved to use WSA for the time that I did but I stopped when I heard it would be going out of support. It's great for mobile games on PC, even if it's pretty resource intensive. I really, REALLY hope they do make this open-source at the end of the day, though sadly I have my doubts.

**Comment by Wduran84** [comment]
check out https://windowsxlite.com/downloads/ it has a version of WSA that uses Google Play Store
